Tests/SourceKitTests/LocalSwiftTests.swift

```diff
@@ -220,6 +220,115 @@ final class LocalSwiftTests: XCTestCase {
     XCTAssertNotEqual(after, selfDot)
   }
 
+  func testFoldingRanges() {
```

Comment:
I should write tests to see how the different capabilities can affect the response, but I wasn't sure how to approach it here.

Reply:
👍 I suggest creating a new test class that duplicates the TestServer setup from this class, but moves the InitializeRequest out of `setUp` and into the individual test methods.  It can just go at the bottom of this file for now, and if it grows we can move it later.

Reply:
That did the trick perfectly. Thanks!